### PR TITLE
Update test/README

### PR DESCRIPTION
**Pull request by frederikwh**
Changed a bad reference: test/Makefile -> test/build.info

##### Checklist

- [x] documentation is added or updated

### CODE REVIEW

**Comment by levitte** [comment]
@openssl/omc will have to decide if this can be deemed trivial or not.  I'm uncertain.

**Comment by tmshort** [comment]
It's a README change of a single sentence. If this isn't considered a trivial change, I don't know what is!

**Comment by levitte** [comment]
Yeah, I know, and if I only listen to my heart, I would completely agree.  However, we all differ, and if we consider originality, and non-copyrightable  is sometimes talked about in terms of "we can't imagine any other way to write this", and that's hardly the case in prose, I don't know what to think any more...

**Comment by t-j-h** [comment]
I'm happy to consider this trivial myself.

**Comment by vdukhovni** [comment]
I don't see how it could be materially more trivial.  My view is no CLA required.

**Comment by frederikwh** [comment]
hi,

Can somebody help me submitting a CLA? It might be useful in the future to have it done.

**Comment by paulidale** [comment]
The CLA information is at [https://www.openssl.org/policies/cla.html](https://www.openssl.org/policies/cla.html).

You will want the individual version: [ICLA](https://www.openssl.org/policies/openssl_icla.pdf).

**Comment by paulidale** [comment]
If you are doing OpenSSL development for your employer, they'll need to fill out the corporate CLA as well.

**Comment by frederikwh** [comment]
> The CLA information is at https://www.openssl.org/policies/cla.html.
> 
> You will want the individual version: [ICLA](https://www.openssl.org/policies/openssl_icla.pdf).

Done. Thanks.

**Comment by paulidale** [comment]
For just this PR, we need "CLA: trivial" on a line by itself in the commit message.  The easiest way to do this is: `git commit --amend` followed by `git push -f`.

**Comment by frederikwh** [comment]
> For just this PR, we need "CLA: trivial" on a line by itself in the commit message. The easiest way to do this is: `git commit --amend` followed by `git push -f`.

Done. What is the status of my CLA submission?

**Comment by paulidale** [comment]
@levitte a re-review due to an additional change.

File: test/README [hunk 1]
```diff
@@ -100,7 +101,7 @@ to modify the include paths and source files if you don't want to use the
 basic test framework:
 
     SOURCE[{name}]={name}.c
-    INCLUDE[{name}]=.. ../include
+    INCLUDE[{name}]=.. ../include ../app/include
```
Comment (levitte) [inline test/README]: `app` should be `apps`

**Comment by mattcaswell** [comment]
> Done. What is the status of my CLA submission?

It has been registered - thanks!

File: test/README (hunk 1 above)
Comment (frederikwh) [reply test/README]: Well spotted!

**Comment by paulidale** [comment]
Merged, thanks.

**Pull request closed without merging**